rllib/utils/tests/test_actor_manager.py

```diff
@@ -262,6 +263,28 @@ def test_async_calls_get_dropped_if_inflight_requests_over_limit(self):
 
         manager.clear()
 
+    def test_healthy_only_works_for_list_of_functions(self):
```

Comment:
Also test  a case where number of functions are you actors are different.

Reply:
done